Dockerfile

```diff
@@ -1,24 +1,28 @@
 # https://github.com/hyrise/hyrise/wiki/Docker-Image
 
-FROM ubuntu:19.04
+FROM ubuntu:19.10
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update \
+    && apt-get install -y software-properties-common \
+    && add-apt-repository -y ppa:mhier/libboost-latest \
+    && apt-get update \
     && apt-get install -y \
         autoconf \
         bash-completion \
         bc \
         ccache \
-        clang-8 \
-        clang-format-8 \
-        clang-tidy-8 \
+        clang-9 \
+        clang-format-9 \
+        clang-tidy-9 \
         cmake \
         curl \
         g++-9 \
         gcc-9 \
         gcovr \
         git \
         graphviz \
-        $(apt-cache search --names-only '^libboost1.[0-9]+-all-dev$' | sort | tail -n 1 | cut -f1 -d' ') \
+        libboost1.70-dev \
+        libhwloc-dev \
```

Comment:
Add to dependency table? Both, libhwloc and the new boost version?

Reply:
Yepp. One day I will upgrade a package and remember all the places where we list them :/

Reply:
We need a script for that :)